### PR TITLE
Added event detail view, redirect on event create

### PR DESCRIPTION
**Pull request by ekong2**

### CODE REVIEW

File: client/public/js/components/EventDetail.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+var React = require('react'),
+    EventDetailActions = require('../actions/EventDetailActions'),
+    EventStore = require('../stores/EventStore'),
+    EventDetailActions = require('../actions/EventDetailActions');
```
Comment (stevenshuhyow) [inline client/public/js/components/EventDetail.js]: repeat

Comment (stevenshuhyow) [inline client/public/js/components/EventDetail.js:3]: repeat

**Comment by stevenshuhyow** [comment]
cool :100:

**Comment by stevenshuhyow** [comment]
need some documentation

**Comment by kevhuang** [comment]
GOOD

**Pull request merged**